src/routes/recipient/index.js

```diff
@@ -2,10 +2,14 @@ import express from 'express';
 import {
   getRecipient,
   searchRecipients,
+  getGoalsByRecipient,
+  changeRecipientGoalStatus,
 } from './handlers';
 
 const router = express.Router();
 router.get('/search', searchRecipients);
 router.get('/:recipientId', getRecipient);
+router.get('/goals/:recipientId', getGoalsByRecipient);
+router.put('/goals/:goalId', changeRecipientGoalStatus);
 
```

Comment:
just looking at this namespacing, looks a little weird... what is supposed to happen if a goal is id 158 and a recipient is id 158? 

Looking at it this way, I think the get should be /:recipientId/goals. What do you think?

The second route also doesn't denote changing status alone, so I guess we need to decide if we will need additional put routes to change other parts (like /goals/status/:goalId) or handle it more like the activity reports with a more generic put route

Reply:
As we discussed update to: /api/recipient/goals/14185/changeStatus.